vendor/assets/stylesheets/dvl/components/tooltips.scss

```diff
@@ -10,7 +10,6 @@
   min-width: 40px;
   transition: transform 0.25s $easeOutBack, opacity 120ms ease-out;
   // Override styles on parent links
-  line-height: $lineHeight;
```

Comment:
I'm pretty sure this was important to have in here, as per the comment on line 12.